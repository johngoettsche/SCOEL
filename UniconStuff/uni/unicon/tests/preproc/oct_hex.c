/* test treatment of octal and hexidecimal constants by string concatentation */

"\001" "1",
"\01" "1",
"\18" "1",
"\01" "\01",
"\01" "8",
"abc\2d\0" "0",
"\77" "0\1",

"\xff" "f\xff" "g",
"\x0a\x0" "0\x0" "x"
