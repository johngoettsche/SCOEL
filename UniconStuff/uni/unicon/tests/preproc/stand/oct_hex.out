#line 2 "oct_hex.c"

"\0011",
"\0011",
"\181",
"\01\01",
"\018",
"abc\2d\0000",
"\0770\1",

"\377f\xffg",
"\x0a\0000\x0x"
